data/json/construction.json

```diff
@@ -17,6 +17,15 @@
         "post_special" : "done_deconstruct"
     },{
         "type" : "construction",
+        "description" : "Take Down Simple Furniture",
```

Comment:
I'd recommend sticking with "Deconstruct" rather than "Take Down" in the description title.

To me, "Take Down" seems to connote that the furniture will still be in one piece after the action. Plus, since this action has the same result as the regular "Deconstruct Furniture" action, sticking with the same naming convention would probably be helpful to convey to players what this will do.